test/unit/refresh.js

```diff
@@ -1296,6 +1296,57 @@ describe('swiftserver:refresh', function () {
 
   });
 
+  describe('Generated a bff application with alert notification for bluemix', function() {
```

Comment:
Minor: The description says "bff application" (as does the watson conversation one above), but the spec doesn't specify the options for a bff. It looks like a "Basic application".